screenpy_selenium/actions/save_console_log.py

```diff
@@ -49,12 +49,16 @@ class SaveConsoleLog:
     """
 
     attach_kwargs: dict | None
-    path: str
-    filename: str
+    path: Path
+
+    @property
+    def filename(self) -> str:
+        """The filename only, no path."""
+        return self.path.name
 
     def describe(self) -> str:
         """Describe the Action in present tense."""
-        return f"Save browser console log as {self.filename}"
+        return f"Save browser console log as {self.path.name}"
```

Comment:
I'm a little confused; after adding the property `filename` why change this line?